config/custom_items.txt

```diff
@@ -815,4 +815,14 @@ ckey: achroknight
 character_name: Aaron Ehrenhof
 item_path: /obj/item/toy/torchmodel
 item_desc: This is a replica of the SEV Torch, in 1:250th scale, on a handsome wooden stand. Small lights blink on the hull and at the engine exhaust. Engraved in a small plaque on the stand is "A. Ehrenhof".
-}
\ No newline at end of file
+}
+{
+ckey: sweedle
+character_name: Max Adams
+item_path: /obj/item/weapon/clipboard
+item_name: legal pad
+item_icon: adamslegal
+item_desc: A blast from the past, low-quality paper bound to a cardboard back. Its pages are no doubt filled with half-complete drafts of essays and official statements. Its owner's name is scrawled across the top in barely legible writing: ADAMS
+req_titles: SolGov Representative
+req_access: 1
```

Comment:
Ditto as above

Reply:
Sorry, I'm not sure what you mean by "setting this"? It seemed to work fine in testing.

Reply:
This will only spawn if you have security access & your job is SolGov Representative.

SolGov Representative might change and lose security access at some point, and if it does, you'll lose access to this item. It's best to just remove `req_titles` and `req_access` whenever it won't be an issue with you getting an item you shouldn't have.

Since this is just a clipboard, there's no issue with you having this item on any job.